implentio/ui-toolkit
# IUT-38 Create Collapse Icons and Key Icon

## Description
There are two components made in this PR: a _collapse_ and _expand_ icon as well as a _key_ icon.
The collapse and expand icons have one prop: `expand` which is a **boolean** to set the direction of the arrow in the icon.
The key icon has one prop: `active` which has 3 possible types of **default**, **active**, and **disabled**.

## Link to JIRA ticket
[IUT-38 Create Collapse and Key Components ](https://implentio.atlassian.net/jira/software/projects/IUT/boards/10?selectedIssue=IUT-38)

## Applicable Screenshots
Include all screenshots that help someone to understand what you made. Include before/after shots if applicable. Include mobile responsive shots if applicable.

## Testing Instructions
- Fetch and checkout my branch `git fetch && git checkout feat/IUT-38/CollapseAndKey`
- run `npm run storybook` in the terminal
- Check that the style of the icons changes when the prop values change

## Checklist:
Check what is applicable by putting an 'x' in between the []:
